### DISCUSSION TITLE
API rule to check value against specified (in-rule) list

### DISCUSSION THREAD

**Discussion by kendallroth** [pocketbase/pocketbase#5262] [Q&A] [answered]
If I have an `user` entity that has an enum `role` field, is there a way to quickly check access against a subset of the roles, rather than comparing all individually? For example, something like `@request.auth.role IN ["admin", "manager"]`, rather than comparing the auth role against each individual target role with an `OR`?

```
// Not ideal
@request.auth.collectionName = "userAdmin" && (@request.auth.role = "admin" || @request.auth.role = "manager")

// Example (doesn't work)
@request.auth.collectionName = "userAdmin" && (@request.auth.role ?= ["admin", "manager"])
```

**Comment by ganigeorgiev** [comment, marked as answer]
No, there is no support for this at the moment.

**Reply by kendallroth** [reply]
Good to know, thanks! Would this be something that could be added in the future, or would it cause issues with the rule parsing?
